src/core/service.c

```diff
@@ -2134,6 +2160,9 @@ static int service_serialize(Unit *u, FILE *f, FDSet *fds) {
 
         unit_serialize_item(u, f, "forbid-restart", yes_no(s->forbid_restart));
 
+        if (s->watchdog_override_usec)
+               unit_serialize_item_format(u, f, "watchdog_override_usec", USEC_FMT, s->watchdog_override_usec);
```

Comment:
all other serialized props use "-" instead of "_" as word separator, please do the same.